File: Task1.py
stroka = input("Enter the string: ")
vowels = 'aeyuoiAEYUOI' 
consonants = 'bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ' 
num_vowels = 0
num_consonants = 0
for i in stroka: 
    if i in vowels: 
        num_vowels += 1
for j in stroka: 
    if j in consonants: 
        num_consonants +=1
if (num_vowels > num_consonants):
  print("Yes")
if (num_vowels < num_consonants):
  print("No")
if (num_vowels == num_consonants):
  print("Identical")

File: Task1.js
let stroka = prompt("Enter the string: ");
let vowels = 'aeyuoiAEYUOI';
let consonants = 'bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ';
let num_vowels = 0;
let num_consonants = 0;

for (let i of stroka) {
  if (vowels.includes(i)) {
    num_vowels += 1;
  }
}

for (let j of stroka) {
  if (consonants.includes(j)) {
    num_consonants += 1;
  }
}

if (num_vowels > num_consonants) {
  console.log("Yes");
} else if (num_vowels < num_consonants) {
  console.log("No");
} else {
  console.log("Identical");
}
